Resources/Locale/en-US/components/space-heater-component.ftl

```diff
@@ -0,0 +1,13 @@
+﻿comp-space-heater-ui-thermostat = Thermostat :
+comp-space-heater-ui-toggle = Toggle
+comp-space-heater-ui-mode = Mode
+comp-space-heater-ui-status-disabled = Off
+comp-space-heater-ui-status-enabled = On
+comp-space-heater-ui-increase-temperature-range = +
+comp-space-heater-ui-decrease-temperature-range = -
+
+comp-space-heater-mode-Auto = Auto
+comp-space-heater-mode-Heat = Heat
+comp-space-heater-mode-Cool = Cool
+
+comp-space-heater-unanchored = The device isn't anchored
```

Comment:
```suggestion
comp-space-heater-unanchored = The device is not anchored.
```
Even better is if you can pass the `{$device}` in as a localized string and then change it to `THE{$device} is not anchored.` so that this can be used for other devices, too.